### PR TITLE
disable revision history before running DMS migration

### PR DESCRIPTION
**Pull request by taroface**
Instruct users to disable backup revision history before running DMS (and re-enable after migration completes).

### CODE REVIEW

**Review by otan — APPROVED** [review]

File: v21.2/aws-dms.md
Comment (kathancox) [inline v21.2/aws-dms.md:27]: Optional: Might be helpful to include a link to this page take-backups-with-revision-history-and-restore-from-a-point-in-time.html

**Review by kathancox — APPROVED** [review]
LGTM, one optional thing!

File: v21.2/aws-dms.md
Comment (awoods187) [inline v21.2/aws-dms.md:27]: DMS lets you both one-time migrate all data or replicate ongoing changes. This guidance seems fine for the one-time migration but I wonder about the ongoing replication as that could last weeks depending upon your migration rules. We should check with @otan but I believe that we need both the initial migration and ongoing replication to be complete. If so, we may want to specify. On the other hand, maybe the language as it is currently worded is sufficient since it says before and after you migrate.

**Review by awoods187 — APPROVED** [review]
Overall LGTM

File: v21.2/aws-dms.md
Comment (otan) [reply v21.2/aws-dms.md:27]: Hmm actually, now that you mention it I would say this tip only applies if you are using replication as a part of the migration strategy.

> we need both the initial migration and ongoing replication to be complete

Yeah it'd need to be ongoing.

Comment (taroface) [reply v21.2/aws-dms.md:27]: Do you mean that the tip to disable revision history only applies when someone is using replication on an ongoing basis, and not when they are using a replication task for a one-time migration?

Comment (otan) [reply v21.2/aws-dms.md:27]: not quite - on AWS DMS, if a users select "migrate existing data and ongoing replication" or "ongoing replication" only then they need this. if they have only "migrate existing data" then they don't need this

Comment (taroface) [reply v21.2/aws-dms.md]: OK, I've attempted to clarify this. PTAL!

**Review by otan — APPROVED** [review]
you bloody legend

**Comment by taroface** [comment]
Thanks to the build breaking, I just noticed that `ALTER BACKUP SCHEDULE` is v22.2 only, and CC isn't on v22.2 until first week of Jan. Unless there is an alternate syntax we can readily document, I think it's best to wait until then to merge this. Or if anyone disagrees, I can amend the recommendation here to "contact Support" and then update it once CC is on v22.2.

**Review by otan — APPROVED** [review]

**Pull request merged**
